homeassistant/components/zha/cover.py

```diff
@@ -120,15 +126,26 @@ def current_cover_position(self) -> int | None:
         """
         return self._current_position
 
+    @property
+    def current_cover_tilt_position(self) -> int | None:
+        """Return the current tilt position of the cover."""
+        return self._tilt_position
+
     @callback
     def async_set_position(self, attr_id, attr_name, value):
         """Handle position update from cluster handler."""
-        _LOGGER.debug("setting position: %s", value)
-        self._current_position = 100 - value
+        _LOGGER.debug("setting position: %s %s %s", attr_id, attr_name, value)
+        if attr_name == "current_position_lift_percentage":
+            self._current_position = 100 - value
+        elif attr_name == "current_position_tilt_percentage":
+            self._tilt_position = 100 - value
+
         if self._current_position == 0:
             self._state = STATE_CLOSED
         elif self._current_position == 100:
             self._state = STATE_OPEN
+        else:
+            self._state = None
```

Comment:
What does this branch add? It seems like it changes the functionality of lift. If it's not required for this PR, let's omit it for now and then address it in a followup PR (probably #99646)